src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -24,16 +24,16 @@ public class JsonAdaptedPersonTest {
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_MOD = "CS!!0!S";
     private static final String INVALID_SKILL = "Àpython";
-
+    private static final String invalidMod = "CS1010";
```

Comment:
I think it would be better to change INVALID_MOD and invalidMod to become INVALID_MODULE, so as to be consistent with the namings.